c2corg_ui/static/js/listswitchaction.js

```diff
@@ -0,0 +1,48 @@
+goog.provide('app.ListSwitchActionController');
+goog.provide('app.listSwitchActionDirective');
+
+goog.require('app');
+
+
+/**
+ * @return {angular.Directive} The directive specs.
+ */
+app.listSwitchActionDirective = function() {
+  return {
+    restrict: 'E',
+    controller: 'appListSwitchActionController',
+    controllerAs: 'switchActionCtrl',
+    templateUrl: '/static/partials/listswitchaction.html'
+  };
+};
+app.module.directive('appListSwitchAction', app.listSwitchActionDirective);
+
+
+/**
+ * @param {app.ListSwitch} appListSwitch service
+ * @constructor
+ * @ngInject
+ */
+app.ListSwitchActionController = function(appListSwitch) {
+
+  /**
+   * @type {app.ListSwitch}
+   */
+  this.listSwitchService = appListSwitch;
+
+  /**
+   * @type {boolean}
+   * @export
+   */
+  this.showList = this.listSwitchService.isList();
```

Comment:
isList() returns whether the display is configured for lists or cards: https://github.com/c2corg/v6_ui/pull/1713/files#diff-1cd444cb2344de44c8f6791257e43f67R37

Or maybe I didn't understand your question?